### ISSUE TITLE
Trill line not read by screen reader

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#34355]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

STEPS:
1.  Add a trill line to a note from the ornament palette.
2. Arrow across the note and notice it reads correctly with the screen reader.
3. Use Alt arrow to move on to the trill line itself and the screen reader says something like "next element" which seems to be the current default when it can't give a name, previously it said blank, other QT versions it said nothing and at one stage said dummy panel on unlabelled elements.  

Currently the trill line is also not showing in Braille.  Most other ornaments are, so maybe it needs this labelled first?

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7.4

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
